geom.js

```diff
@@ -3,17 +3,55 @@ class Rectangle {
     this.length = length;
     this.width = width;
   }
+  square(){
+    if(this.length == this.width){
+    console.log("this is a square")
```

Comment:
It would be helpful if you returned `true` or `false` in this method so that you could do something like this later on...

```js
if(squareInstance.square()){
  // run could that happens if it is a square
}
```